eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -23,9 +23,11 @@
 
      <!-- This is upgraded to latest version in full source-build and can be baselined for repo build -->
     <UsagePattern IdentityGlob="System.Composition*/7.0*" />
-    <UsagePattern IdentityGlob="System.Configuration.ConfigurationManager/7.0*" />
-    <UsagePattern IdentityGlob="System.Diagnostics.EventLog/7.0*" />
-    <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/7.0*" />
+    <UsagePattern IdentityGlob="System.Configuration.ConfigurationManager/8.0*" />
+    <UsagePattern IdentityGlob="System.Diagnostics.EventLog/8.0*" />
+    <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/8.0*" />
+    <UsagePattern IdentityGlob="System.Security.Permissions/8.0*" />
+    <UsagePattern IdentityGlob="System.Windows.Extensions/8.0*" />
```

Comment:
These will also need to have corresponding `PackageVersion` entries, pointing to version properties from eng/Versions.props.

Reply:
Added.